packages/examples/src/appPlayground/main.ts

```diff
@@ -128,6 +131,12 @@ export const runApplicationPlayground = async () => {
     const result = wrapper.getExtensionRegisterResult('mlc-app-playground') as RegisterLocalProcessExtensionResult;
     result.setAsDefaultApi();
 
+    // currently unused
+    await getService(IWorkbenchLayoutService);
+
+    // show explorer and not search
+    await vscode.commands.executeCommand('workbench.view.explorer');
```

Comment:
I'm not sure why it's required :thinking: it would prevent the storage service from being able to restore the last open view

Reply:
So, you are saying, it should not behave as it currently, but maybe it it does because the storage service is not used?

Reply:
I see no reason for the search service to be displayed by default :thinking:

The storage service not being used just prevents it from storing and restoring the active view, it shouldn't have any impact on the default view on a fresh new browser

Reply:
When the storage service is used. The positions are kept in local storage, but the search is always displayed, so there seems to be another issue:
![image](https://github.com/user-attachments/assets/5bfe5bd5-7834-4114-ab97-1399aab800a4)

Reply:
That weird :thinking:  you can try to play with the `defaultLayout.views` construct option :thinking: 

Maybe there is a bug or a race condition in the code

It probably has something to do with https://github.com/CodinGame/monaco-vscode-api/blob/c228c61f6dbcfebeeffa6769373ff356abacc0e3/src/service-override/views.ts#L427-L471, you can try debugging in there?

Reply:
`defaultLayout.views` allows to select which view should be displayed at first, I'm curious what happens it you force the explorer view

Reply:
Ok, it doesn't change the behaviour

Reply:
Ok so there is probably something that changes it later...

Reply:
Yes, that seems to be the case.

Reply:
Can you have a look at some point. I [suggested the workaround for now](https://github.com/TypeFox/monaco-languageclient/issues/815#issuecomment-2573328837).